interactive/st_utils.py

```diff
@@ -0,0 +1,115 @@
+import streamlit as st
+
+from loguru import logger
+from interactive import GRID_NROW_NUMBER
+
+
+class StatesManager:
+    """
+    Class defining list of available application states.
+    """
+
+    def __init__(self):
```

Comment:
i guess you can move that logger info into initialize states, initializing object without any class members are hardly seen